### DISCUSSION TITLE
XLS batch modification

### DISCUSSION THREAD

**Discussion by duemme** [saulpw/visidata#903] [Q&A]
Hello,
I would like to use VISIDATA to modify a series of XLS files that have the same internal structure but different data.
I would like to pass to VISIDATA the file and some variables like "names of colomuns", "fields content" and make VISIDATA delete some rows, delete some columns, change columns names, add new columns and fill in rows with a value, substitute row values.
I have little and basic knowledge (close to zero) on programmin and scripting and I didn't find info or documentation about this topic.
I look forward to getting some help from the comunity.
Thanks

**Comment by aborruso** [comment]
Ciao @duemme ,
it's possible to automate this kind of stuff. Could you share some example input and add some example tasks?
